### DISCUSSION TITLE
Looks like csb domain is being blocked again and direct links to csb based samples fail

### DISCUSSION THREAD

**Discussion by johnman** [codesandbox/codesandbox-client#4370] [General] [answered]
Hi,

This issue has come up before. It looks like the csb domain is being blocked (possibly because someone misused a csb based sandbox).

That makes direct links to demos fail.

Can someone please look into this .

Thanks,

John

**Comment by garethx** [comment, marked as answer]
Please see the update in https://github.com/codesandbox/codesandbox-client/issues/3751
